keyboards/ergodash/rev2/rev2.c

```diff
@@ -30,6 +30,7 @@ void matrix_init_kb(void) {
 	matrix_init_user();
 };
 
+__attribute__ ((weak))
```

Comment:
Actually, this function should be removed.

All it's doing is the "shutdown sound", and it's actually overriding the globally configured one.